src/Rocketlet.ts

```diff
@@ -83,8 +84,7 @@ export abstract class Rocketlet {
      * extended by this Rocketlet have one or multiple Rocketlet data items?
      *
      */
-    // TODO: Discuss what the purpose of this is, commenting out for now.
-    // public abstract getRocketChatAssociation(): IRocketChatAssociation;
+    public abstract getRocketChatAssociations(): Array<IRocketChatAssociation>;
```

Comment:
I sind't see how implementing different interfaces would help here, so I just enhanced this metadata-retrieval to support - literally - multiple associations